services/scabbard/libscabbard/src/client/mod.rs

```diff
@@ -29,18 +29,69 @@ use super::protocol::SCABBARD_PROTOCOL_VERSION;
 
 pub use error::ScabbardClientError;
 
+/// Builder for building a ScabbardClient.
```

Comment:
In doc comments, it's general best to put names of code items in backticks so they are formatted as code. When referencing an item like this, it's even better to add a link like this: ``[`ScabbardClient`](crate::client::ScabbardClient)``.

Reply:
Fixed